python/dalex/dalex/fairness/_group_fairness/plot.py

```diff
@@ -221,7 +220,7 @@ def plot_metric_scores(fobject,
     data['subgroup_numeric'] = [subgroup_tick_dict.get(sub) for sub in data.subgroup]
     data = data.reset_index(drop=True)
     data.subgroup_numeric = data.subgroup_numeric + pd.Series([label_tick_dict.get(lab) for lab in data.label])
-
+    data.iloc[:, 1:]
```

Comment:
is this line necessary?

Reply:
No, it was left after debugging. Thanks for catching that